editor/src/key-listener/change-bounds.ts

```diff
@@ -0,0 +1,44 @@
+import { SelectionService } from '@eclipse-glsp/client/lib/features/select/selection-service';
+import { KeyListener, Action, SModelElement, BoundsAware } from 'sprotty';
+import { matchesKeystroke } from 'sprotty/lib/utils/keyboard';
+import { inject } from 'inversify';
+import { GLSP_TYPES, isNonRoutableSelectedMovableBoundsAware, toElementAndBounds } from '@eclipse-glsp/client';
+import { ChangeBoundsOperation, ElementAndBounds } from '@eclipse-glsp/protocol';
+import { IvyGridSnapper } from '../diagram/snap';
+
+export class ChangeBoundsKeyListener extends KeyListener {
```

Comment:
I would change the name of this keylistener. Sure it will fire changebound operations, but technically it moves elements. So I would vote for MoveElementKeyListener or something like that.